### DISCUSSION TITLE
Need help validating a possible bug affecting homebrew builds of DOSBox Staging

### DISCUSSION THREAD

**Discussion by almeath** [Homebrew/discussions#2240] [Everyday usage]
I am looking for help from anyone who is still on macOS Mojave (10.14.6) and who builds their own DOSBox Staging using Homebrew (as it is not available pre-built for this macOS version). I am told there could be a potential bug "on line 27 of Brew's ruby manifest", but I am not sure if this issue is only affecting my own system. The description of the issue is available here:

https://github.com/dosbox-staging/dosbox-staging/issues/1270

I am not sure how to properly use the gist-logs or config/doctor output, so before I delve into that I was hoping someone could validate whether they also experience the same problem on their own Mojave install, i.e. when you run "brew install dosbox-staging" you end up with a DOSBox Staging build that is lacking functioning MT-32 emulation support.

**Comment by carlocab** [comment]
This doesn't seem like a bug -- you're trying to use a feature that's been explicitly disabled. Enabling the feature (as long as it does not introduce too many new dependencies) seems reasonable, however.

Can you check if replacing `-Duse_mt32emu=false` with `--force-fallback-for=mt32emu` and rebuilding `dosbox-staging` from source fixes your problem for you? If it does, it'd be great if you can open a pull request so we can fix this for everyone using the formula.

Also, as context for why that flag was added: https://github.com/Homebrew/homebrew-core/pull/80510#discussion_r663435453 It seems the build fails without it. Perhaps using `--force-fallback-for=mt32emu` will fix that, though.
